modules/acquia_cms_headless/composer.json

```diff
@@ -68,9 +68,6 @@
             "drupal/decoupled_router": {
                 "3111456 - Unable to resolve path on node in other language than default": "https://www.drupal.org/files/issues/2022-11-30/decouple_router-3111456-resolve-language-issue-58.patch"
             },
-            "drupal/next": {
```

Comment:
@apathak18 - I don't see all changes from https://www.drupal.org/files/issues/2023-02-08/next-module-unpublished-preview-3340189.patch in this file specially the css one. it is something we don't need?
```
diff --git a/css/next.site_preview.iframe.css b/css/next.site_preview.iframe.css
index 0803bcb..1ee5e39 100644
--- a/css/next.site_preview.iframe.css
+++ b/css/next.site_preview.iframe.css
@@ -33,7 +33,7 @@
   color: #325e1c;
 }
 
-.next-site-preview-toolbar .operations .live-link .button {
+.next-site-preview-toolbar .operations .live-link .preview-link .button {
   margin-top: 0;
   margin-bottom: 0;
 }
```

Reply:
Not required @chandan-singh7929 
cc: @vishalkhode1